journalbeat/README.md

```diff
@@ -0,0 +1,27 @@
+# journalbeat
+
+Journablbeat is a lightweight shipper for journald logs with Elasticsearch.
+
+## Maintainers
+
+The Habitat Maintainers (humans@habitat.sh).
+
+## Type of package
+
+Service package.
+
+## Usage
+
+```
+hab svc load core/journalbeat
+```
+
+Metricbeat will run with the default configuration which puts logs at `{{svc_var_path}}/log/*.log`. The default configuration may not match your needs, and you can replace all of the configuration by providing a user.toml.
```

Comment:
I think this should read `Journalbeat`.

Reply:
The dangers of using another plan as a template
Fixed